src/gs-shell.c

```diff
@@ -782,6 +788,37 @@ gs_shell_reload_cb (GsPluginLoader *plugin_loader, GsShell *shell)
 	}
 }
 
+static void
+gs_shell_reload_and_select_removable_media_category (GsShell *shell,
+						     gboolean reload_overview,
+						     gboolean select_category,
+						     const gchar *category_name)
+{
+	/* using NULL for this below indicates it should not be selected */
```

Comment:
This seems like a thing that may cause trouble down the road. Should we have a simple `reload_overview` that selects no category? Or is this functionality being used more times?

Reply:
Yeah, splitting it seems like the cleanest option. There's trivial code duplication but it's three lines and one of those is just fetching `priv`, etc. This helps clean up the code a decent amount and lets me cut out one of the commits I described above.